test/checks/aria/required-children.js

```diff
@@ -49,6 +49,12 @@ describe('aria-required-children', function () {
 		assert.isTrue(checks['aria-required-children'].evaluate.call(checkContext, node));
 	});
 
+	it('should pass a native input with role comboxbox when missing child is role textbox', function () {
+		fixture.innerHTML = '<input type="text" role="combobox" aria-owns="listbox" id="target"></div><p role="listbox" id="listbox">Nothing here.</p>';
```

Comment:
Good to know! I assume that change had already been made to all other tests in your Shadow DOM branch. Thanks for bringing this test into the shadowy future.